src/media/js/header_footer.js

```diff
@@ -20,9 +20,16 @@ define('header_footer',
 
             z.body.attr('data-mkt-nav--enabled', true);
 
+            if (window.innerWidth >= 800) {
```

Comment:
we've been using https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia in previews.js, not sure if that's necessary or better